mono_repo/lib/src/runner.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:args/command_runner.dart';
+
+import 'commands/check.dart';
+import 'commands/init.dart';
+import 'commands/presubmit.dart';
+import 'commands/pub.dart';
+import 'commands/travis.dart';
+
+const _name = 'mono_repo';
```

Comment:
What's the upside to creating these consts – vs just plugging the literals in the `super` call?

Reply:
Not significant. Done.